bin/export.py

```diff
@@ -321,6 +321,13 @@ def update_problems_yaml(problems):
                 }
             )
 
+    if colors:
+        for d, c in zip(data, colors):
+            color = ('' if c.startswith('#') else '#') + c
```

Comment:
Simply `c or '#'+c` is clearer imo

Reply:
That does not check whether `c` starts with a `#`. The expectation is that `c` is a string of the shape `#rrggbb` or `rrggbb`, and this normalises `c` to always start with `#`